.github/workflows/visual-regression.yml

```diff
@@ -0,0 +1,58 @@
+name: Visual regression with Netlify
```

Comment:
I think this is a kind of regression tests to detect the degradation of visualization functions.
So, how about adding `tests` to the name?
```suggestion
name: Visual regression tests with Netlify
```

Reply:
Agreed. I revised the code.

Reply:
Thank you.